test/built-ins/Promise/all/resolve-element-function-name.js

```diff
@@ -2,6 +2,7 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+esid: sec-promise.all-resolve-element-functions
 es6id: 25.4.4.1.2
```

Comment:
When you add an `esid`, you should remove the `es6id`

Reply:
Will do, thank you.
65c230a